src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -17,6 +18,31 @@ public String toString() {
         return columnName;
     }
 
-    public static final String[] COLUMN_NAME_ARRAY =
-            Stream.of(values()).map(AllelicCountTableColumns::toString).toArray(String[]::new);
+    public static final String[] BASIC_COLUMN_NAME_ARRAY = {
```

Comment:
Also I wonder whether we should also add a more general TableColumnCollection.containsAll(Object ... clumns), that internally would translate non-string object into strings using toString so that you don't have to do string reformatting here... To much work for this pull-request but something to think about in the future.